File: RepoTransBench/source_projects/Java/xiaofeidev_RoundWidget/lib_round/src/test/java/io/github/xiaofeidev/round/RoundImageViewPublicTest.java
package io.github.xiaofeidev.round;

import android.content.Context;
import android.util.AttributeSet;

import org.junit.Test;
import org.junit.runner.RunWith;
import org.robolectric.RobolectricTestRunner;
import org.robolectric.RuntimeEnvironment;

import static org.junit.Assert.*;

@RunWith(RobolectricTestRunner.class)
public class RoundImageViewPublicTest {

    @Test
    public void constructor_and_init_minimal_public() {
        Context context = RuntimeEnvironment.getApplication();
        RoundImageView view = new RoundImageView(context);
        assertNotNull(view);
        assertEquals(RoundImageView.STROKE_MODE_PADDING, 0);
        assertEquals(RoundImageView.STROKE_MODE_OVERLAY, 1);
        assertTrue(view instanceof RoundImageView);
        assertNotNull(view.getRadiusList());
    }

    @Test
    public void constructor_with_attrs_public() {
        Context context = RuntimeEnvironment.getApplication();
        AttributeSet attrs = null;
        RoundImageView view = new RoundImageView(context, attrs);
        assertNotNull(view);
        assertNotNull(view.getRadiusList());
    }

    @Test
    public void testSetAndGetRadius_public() {
        Context context = RuntimeEnvironment.getApplication();
        RoundImageView view = new RoundImageView(context);
        view.setRadius(12f);
        assertEquals(12f, view.getRadius(), 0f);
        view.setTopRightRadius(7f);
        assertEquals(7f, view.getTopRightRadius(), 0f);
    }

    @Test
    public void testFillRadiusReflects_public() {
        Context context = RuntimeEnvironment.getApplication();
        RoundImageView view = new RoundImageView(context);
        view.setRadius(9f);
        view.setTopLeftRadius(2.2f);
        view.setTopRightRadius(3.3f);
        view.setBottomLeftRadius(4.4f);
        view.setBottomRightRadius(5.5f);
        view.fillRadius();
        float[] list = view.getRadiusList();
        assertEquals(2.2f, list[0], 0.0001f);
        assertEquals(3.3f, list[2], 0.0001f);
        assertEquals(4.4f, list[6], 0.0001f);
        assertEquals(5.5f, list[4], 0.0001f);
    }
}

File: RepoTransBench/target_projects/Java/C#/xiaofeidev_RoundWidget/public_tests/RoundImageViewPublicTest.cs
using Xunit;
using RoundWidget;

namespace RoundWidget.Tests.Public
{
    public class RoundImageViewPublicTest
    {
        [Fact]
        public void Constructor_And_Init_Minimal_Public()
        {
            var view = new RoundImageView();
            Assert.NotNull(view);
            Assert.Equal(0, RoundImageView.STROKE_MODE_PADDING);
            Assert.Equal(1, RoundImageView.STROKE_MODE_OVERLAY);
            Assert.IsType<RoundImageView>(view);
            Assert.NotNull(view.getRadiusList());
        }

        [Fact]
        public void Constructor_With_Attrs_Public()
        {
            var view = new RoundImageView();
            Assert.NotNull(view);
            Assert.NotNull(view.getRadiusList());
        }

        [Fact]
        public void TestSetAndGetRadius_Public()
        {
            var view = new RoundImageView();
            view.setRadius(12f);
            Assert.Equal(12f, view.getRadius(), 4);
            view.setTopRightRadius(7f);
            Assert.Equal(7f, view.getTopRightRadius(), 4);
        }

        [Fact]
        public void TestFillRadiusReflects_Public()
        {
            var view = new RoundImageView();
            view.setRadius(9f);
            view.setTopLeftRadius(2.2f);
            view.setTopRightRadius(3.3f);
            view.setBottomLeftRadius(4.4f);
            view.setBottomRightRadius(5.5f);
            view.fillRadius();
            float[] list = view.getRadiusList();
            Assert.Equal(2.2f, list[0], 4);
            Assert.Equal(3.3f, list[2], 4);
            Assert.Equal(4.4f, list[6], 4);
            Assert.Equal(5.5f, list[4], 4);
        }
    }
}